cmd/operator/main.go

```diff
@@ -193,32 +193,38 @@ func run(fs *flag.FlagSet) int {
 		return 0
 	}
 
-	logger, err := logging.NewLogger(logConfig)
+	logger, err := logging.NewLoggerSlog(logConfig)
+	if err != nil {
+		stdlog.Fatal(err)
+	}
+
+	// We're currently migrating our logging library from go-kit to slog.
+	// The go-kit logger is being removed in small PRs. For now, we are creating 2 loggers to avoid breaking changes and
+	// to have a smooth transition.
+	goKitLogger, err := logging.NewLogger(logConfig)
 	if err != nil {
 		stdlog.Fatal(err)
 	}
 
 	if err := cfg.Gates.UpdateFeatureGates(*featureGates.Map); err != nil {
-		level.Error(logger).Log("error", err)
+		logger.Error("", "error", err)
```

Comment:
Interesting thing with slog. Looks like it will **always** have the `msg` field, where go-kit/log doesn't require that.

An empty message will still be logged, as seen in this [playground](https://go.dev/play/p/46QcymutNGw). Not that this is a problem, but we're introducing a new field to all places where `msg` isn't filled with something

Reply:
<img width="724" alt="image" src="https://github.com/user-attachments/assets/3acd20c4-0d98-48d4-b31b-ec56fca0da12">